cmd/lakectl/cmd/auth.go

```diff
@@ -12,17 +12,17 @@ import (
 )
 
 var userCreatedTemplate = `{{ "User created successfully." | green }}
-ID: {{ .ID | bold }}
+ID: {{ .Id | bold }}
```

Comment:
This is unfortunate, but can we tweak the codegen to generate ID?  The strict-ish [standard for initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) says:

> This rule also applies to "ID" when it is short for "identifier" (which is pretty much all cases when it's not the "id" as in "ego", "superego"), so write "appID" instead of "appId".
>
> Code generated by the protocol buffer compiler is exempt from this rule. Human-written code is held to a higher standard than machine-written code.

So:
1. It _is_ required, but...
2. Other robots can claim an exemption so as not to be discriminated vs. Googley tools.

But it still looks odd in Go code, and it *will* confuse us.  I'd be much happier with tweaking our input openAPI spec to generate an `ID` in Go -- if possible.  And preferably before we have other client code that depends on this style.

Reply:
Agree, but it will require a fork and PR for oapi-codegen